mlir/lib/Dialect/Tosa/Transforms/TosaFolders.cpp

```diff
@@ -1266,6 +1266,121 @@ struct TosaFoldConstantMaximum
   }
 };
 
+template <typename BaseType>
+DenseElementsAttr padType(ShapedType inputType, ElementsAttr inputValues,
+                          DenseElementsAttr paddings,
+                          std::optional<DenseElementsAttr> padConstValue,
+                          ShapedType outputType, BaseType zero) {
+  BaseType padConst(zero);
+  if (padConstValue.has_value())
+    padConst = padConstValue.value().getSplatValue<BaseType>();
+
+  auto values = inputValues.getValues<BaseType>();
+  auto paddingVals = paddings.getValues<int64_t>();
```

Comment:
Interesting, the spec says int32 but the op definition in TableGen says int32 or int64 :D

Reply:
True!